### PR TITLE
Refs [Task][39294] Task 2 Create Footer -  [Update] Lặp cart-item

### PR DESCRIPTION
**Pull request by cntt2-vyho**

### CODE REVIEW

**Comment by cntt2-vyho** [comment]
https://edu-redmine.sun-asterisk.vn/issues/39294

**Comment by cntt2-vyho** [comment]
image

**Comment by cntt2-vyho** [comment]
ready

**Comment by cntt2-vyho** [comment]
ready

**Comment by cntt2-vyho** [comment]
ready

**Comment by cntt2-vyho** [comment]
ready

File: pug/partials/header.pug [hunk 1]
```diff
@@ -3,8 +3,39 @@ header.navbar-expand-lg.navbar.bg-light#header
...
+                                .ms-4
+                                    h6.fw-bold.text-uppercase.text-black.mb-0 Áo sơ mi nam
+                                    p.fs-sm.text-gray-700.mb-0.text-primary.text-thirsty 120.000 đ
+                        .list-group.list-group-flush.mb-3
+                            a.list-group-item
+                                .icon.icon-sm
+                                    img.img-cart-size(src="../assets/images/cart-img.jpg" alt="cart-img")
+                                .ms-4
+                                    h6.fw-bold.text-uppercase.text-black.mb-0 Áo sơ mi nam
+                                    p.fs-sm.text-gray-700.mb-0.text-primary.text-thirsty 120.000 đ
```
Comment (vietnq-1097) [inline pug/partials/header.pug]: Lặp code, cho vào vòng lặp nhé

**Comment by cntt2-vyho** [comment]
ready

**Pull request merged**